sdk/tables/data-tables/test/public/specialCharacters.spec.ts

```diff
@@ -0,0 +1,263 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { odata, TableClient, TableEntityResult, TransactionAction } from "../../src";
+import { assert } from "chai";
+import { createTableClient, recordedEnvironmentSetup } from "./utils/recordedClient";
+import { isNode } from "@azure/test-utils";
+import { isLiveMode, record, Recorder } from "@azure-tools/test-recorder";
+import { Context } from "mocha";
+
+describe("SpecialCharacters", function() {
+  before(function(this: Context) {
+    if (!isLiveMode()) {
+      // Currently the recorder is having issues with the encoding of single qoutes in the
+      // query request and generates invalid JS code. Disabling this test on playback mode
+      // while these issues are resolved
```

Comment:
Agh, is there an issue open to track this? if yes, could you inline a link to it here?